### PR TITLE
Org name resolution on Person.add_membership

### PR DESCRIPTION
**Pull request by patcon**
I know add_membership doesn't get used much, but I needed it for scraping LegistarOrganizationScraper when splitting the people scraper from the org scraper.

Much appreciated for considering the merge!

### CODE REVIEW

**Comment by fgregg** [comment]
I'd go further and remove all the filtering. The specific filtering should go in the city specific views.

**Comment by fgregg** [comment]
Can you add a test?

**Comment by patcon** [comment]
Done and done,

Couldn't get tests working locally, so couldn't test for fail, so might want to give a quick look :)

**Comment by jpmckinney** [comment]
What was the error locally for running tests? What command did you run?

File: pupa/tests/scrape/test_people_org_scrape.py [hunk 1]
```diff
@@ -143,3 +142,12 @@ def test_committee_add_member_name():
     assert get_pseudo_id(c._related[0].person_id) == {'name': 'John Adams'}
     assert c._related[0].organization_id == c._id
     assert c._related[0].role == 'member'
+
+def test_person_add_membership_name():
+    p = Person('Leonardo DiCaprio')
+    p.add_membership('Academy of Motion Picture Arts and Sciences', role='winner', start_date='2016')
+    p._related[0].validate()
```
Comment (jpmckinney) [inline pupa/tests/scrape/test_people_org_scrape.py]: Why is this line necessary? There's no similar line in the `test_committee_add_member_name` test.

Comment (patcon) [inline pupa/tests/scrape/test_people_org_scrape.py]: Just kept it from the orginal add_membership test:
https://github.com/opencivicdata/pupa/blob/master/pupa/tests/scrape/test_people_org_scrape.py#L49-L59

Sorry, I can def remove. I'm embarrassed that I don't understand all the testing pieces yet, but that's probably obvious

Comment (jpmckinney) [inline pupa/tests/scrape/test_people_org_scrape.py]: Perhaps it's needed to resolve the org. Anyway, let's see what happens if removed.

**Comment by patcon** [comment]
ok, did some force-pushed with and without, with some junky field values:

fail with bad value: https://travis-ci.org/opencivicdata/pupa/jobs/110828276#L324
pass with bad value: https://travis-ci.org/opencivicdata/pupa/jobs/110852347

seems the fields might be malformed without. Seems nice to leave in, but maybe it's mixing concerns

**Comment by jpmckinney** [comment]
It just makes me think that the tests may be brittle, but since there's another test with the same pattern, YOLO.

**Comment by patcon** [comment]
haha :nail_care: :boom: :fire_engine:
